csunplugged/topics/management/commands/_UnitPlanLoader.py

```diff
@@ -121,6 +118,13 @@ def load(self):
                         "Lesson"
                     )
 
+                if lesson_data is None:
+                    raise MissingRequiredFieldError(
+                        self.structure_file_path,
+                        ["number"],
+                        "Unit Plan"
+                    )
```

Comment:
could probably combine this with the next error handling section below, something like:
```
lesson_number = lesson_data.get("number", None)
if lesson_data is None or lesson_number is None:
    raise MissingRequiredFieldError(
        self.structure_file_path,
        ["number"],
        "Unit Plan"
    )
```
Just reduces some repeated code, not a big deal though.